acj/api/classlist.py

```diff
@@ -219,6 +221,10 @@ def import_users(import_type, course, users):
         db.session.add(enrolment)
     db.session.commit()
 
+    # wait until user ids are generated before starting background jobs
+    for (user, password) in set_user_passwords:
```

Comment:
One advantage is that the passwords can be generated asynchronously this way. I suppose it will cause less load on the workers to have the only one job though. Will change.

Yes, it needs to be serialisable